setup.cfg

```diff
@@ -116,6 +116,12 @@ windows_platform =
 osx_platform =
     %(macos_platform)s
 
+mypy =
```

Comment:
Minor comment.
Maybe add all mypy and mypy-zope directly to dev.
When I do local twisted development, I find it easier to just create a virtual env with [.dev] and get everything.

But if installing mypy in the other dev env takes a  long time, I am ok with using this separate extra.

Reply:
Yeah mypy and mypy-zope are very tiny wheels I'd be happy to do that

Reply:
@adiroiban ah that doesn't work because mypy needs `all_non_platform` and `dev`

Reply:
Yes... so maybe add `all_non_platform` as part of `dev` :) as when you are doing development you most probably want it all

Reply:
Probably a separate ticket